### PR TITLE
Avoid generating package lock file in sample app

### PR DESCRIPTION
**Pull request by yathindrak**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: samples/asgardeo-choreo-react-express/apps/server/.npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (thivi) [inline samples/asgardeo-choreo-react-express/apps/server/.npmrc]: Add a new empty line.

File: samples/asgardeo-choreo-react-express/apps/client/.npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (thivi) [inline samples/asgardeo-choreo-react-express/apps/client/.npmrc]: Add a new empty line.

File: samples/asgardeo-choreo-react-express/.npmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (thivi) [inline samples/asgardeo-choreo-react-express/.npmrc]: Add a new empty line.

**Review by thivi — APPROVED** [review]

File: samples/asgardeo-choreo-react-express/apps/server/.npmrc (hunk 1 above)
Comment (yathindrak) [reply samples/asgardeo-choreo-react-express/apps/server/.npmrc]: Fixed with https://github.com/asgardeo/asgardeo-auth-express-sdk/pull/31/commits/8c323c357d515c449839e1abf2bf7c000237cb6d

File: samples/asgardeo-choreo-react-express/apps/client/.npmrc (hunk 2 above)
Comment (yathindrak) [reply samples/asgardeo-choreo-react-express/apps/client/.npmrc]: Fixed with https://github.com/asgardeo/asgardeo-auth-express-sdk/pull/31/commits/8c323c357d515c449839e1abf2bf7c000237cb6d

File: samples/asgardeo-choreo-react-express/.npmrc (hunk 3 above)
Comment (yathindrak) [reply samples/asgardeo-choreo-react-express/.npmrc]: Fixed with https://github.com/asgardeo/asgardeo-auth-express-sdk/pull/31/commits/8c323c357d515c449839e1abf2bf7c000237cb6d

**Pull request merged**